compiler/il/OMRResolvedMethodSymbol.cpp

```diff
@@ -157,15 +157,17 @@ OMR::ResolvedMethodSymbol::ResolvedMethodSymbol(TR_ResolvedMethod * method, TR::
    //
    if ((_methodIndex > JITTED_METHOD_INDEX && !_resolvedMethod->isSameMethod(comp->getJittedMethodSymbol()->getResolvedMethod()))
        || comp->isDLT()
-       || (comp->getOption(TR_UseSymbolValidationManager) && comp->compileRelocatableCode()))
+       || comp->compileRelocatableCode())
```

Comment:
Please fix copyright header.

Reply:
Didn't OMR have at one point a copyright header checker? Do you know why it was removed?

Reply:
Never implemented: #2254